### ISSUE TITLE
nbconvert to pdf fails after updating OS X to El Capitan

### ISSUE THREAD

**Issue by Oxtay** [ipython/ipython#8935]
Yesterday I converted one of my notebooks to PDF. This morning I upgraded my OS X to the new El Capitan and trying the same conversion today, I got this message:

`nbconvert failed: command could not be found: pdflatex`

I tried a few different things, including updating ipython from 3.1.x to 4.0.0, installing mistune through pip (because somewhere midway it started complaining about that), and updating homebrew. It seems only the message has changed slightly, but I assume it basically means the same thing:

`nbconvert failed: pdflatex not found on PATH`

Not sure if this is an error related to El Capitan or ipython.

**Comment by crocha700** [comment]
Could you just do  ```$ which pdflatex``` and tell me what you get?

**Comment by Oxtay** [comment]
It actually returns nothings. It seems that it might be related to [issue described here about El Capitan](http://www.tug.org/mactex/mactex-download.html).

On a different note, I may have to create another issue for this or it may have been solved, but I noticed that calling `ipython notebook` with `--profile` and `--color` no longer works. Would you happen to know if there is a solution?

**Comment by shaun10** [comment]
Oxtay, know issue has been unresolved for a month but did you try installing python via brew? then installing jupyter/ipython? From there I would try to install pdflatex and echo, which the path back to us for you python install.

**Comment by shaun10** [comment]
Also looks like Latex and MacTex require symbolic link updates once the download is finished unfortunately.

**Comment by takluyver** [comment]
Are you doing the conversion from the notebook interface? How are you starting the notebook. If you start it from a GUI mechanism in OS X, it won't pick up your changes to `$PATH` in `.bashrc`.

**Comment by takluyver** [comment]
Closing as probably not a bug, feel free to continue discussing.

**Comment by Oxtay** [comment]
I can no longer recreate the issue. 
Thanks.
